x-pack/metricbeat/tests/system/metricbeat.py

```diff
@@ -0,0 +1,95 @@
+import re
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../../../libbeat/tests/system'))
+
+from beat.beat import TestCase
+
+COMMON_FIELDS = ["@timestamp", "beat", "metricset.name", "metricset.host",
+                 "metricset.module", "metricset.rtt", "host.name"]
+
+INTEGRATION_TESTS = os.environ.get('INTEGRATION_TESTS', False)
+
+import logging
+logging.getLogger("urllib3").setLevel(logging.WARNING)
+
+
+class BaseTest(TestCase):
```

Comment:
I would think that this file can inherit from the metricbeat.py file we already have.

Reply:
Actually, I'll need help for this. I have been struggling to find out how system tests works under the hood. In fact, there's not much done in the system tests part. Just a simple copy-paste basically.